sysbuild/CMakeLists.txt

```diff
@@ -93,22 +241,33 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_post_cmake)
   cmake_parse_arguments(POST_CMAKE "" "" "IMAGES" ${ARGN})
   restore_ncs_vars()
 
+  set(SYSBUILD y)
+
   set_property(GLOBAL PROPERTY DOMAIN_APP_APP ${DEFAULT_IMAGE})
 
-  # Run partition manager for each image before running CMake.
-  set(domain ${SB_CONFIG_SECURE_BOOT_DOMAIN})
-  if(DEFINED domain AND NOT domain STREQUAL APP)
-    set(partition_manager_target partition_manager_${domain})
-  endif()
+  include_packaging()
 
-  if(SB_CONFIG_SECURE_BOOT)
+  if(SB_CONFIG_SECURE_BOOT OR SB_CONFIG_MCUBOOT_HARDWARE_DOWNGRADE_PREVENTION)
     include_provision_hex()
   endif()
 
   include(${ZEPHYR_NRF_MODULE_DIR}/cmake/extensions.cmake)
   if(SB_CONFIG_PARTITION_MANAGER)
+    # Run partition manager for each image before running CMake.
     include(${ZEPHYR_NRF_MODULE_DIR}/cmake/sysbuild/partition_manager.cmake OPTIONAL)
   endif()
+
+  # temp hack, load the specifically needed PM setting into a CMake current scope var.
```

Comment:
Added JIRA number, left `temp hack` because that's what it is, and it actually causes things to break, Torsten needs to have a look but is busy with other work at present